README.md

```diff
@@ -52,6 +50,9 @@ grunt.initConfig({
 });
 ```
 
+## Examples & options
```

Comment:
I would remove the heading and in the paragraph below I would say something like:

**grunt-uidocs** is only a grunt adapter for [ui-docs](https://github.com/vieron/ui-docs), check the [complete list of configuration options](https://github.com/vieron/ui-docs#options) and some [examples](https://github.com/vieron/ui-docs#example) about its usage.